src/features/navigation/RootNavigator/RootNavigator.tsx

```diff
@@ -38,7 +45,15 @@ const RootStackNavigator = withWebWrapper(
         <RootStack.Navigator
           initialRouteName={initialRouteName}
           screenOptions={ROOT_NAVIGATOR_SCREEN_OPTIONS}>
-          <RootStack.Screen name="CheatcodesStackNavigator" component={CheatcodesStackNavigator} />
+          {env.FEATURE_FLIPPING_ONLY_VISIBLE_ON_TESTING ? (
+            <RootStack.Screen name="CheatcodesStackNavigator">
+              {() => (
+                <Suspense fallback={<LoadingPage />}>
+                  <CheatcodesStackNavigator />
+                </Suspense>
+              )}
```

Comment:
plus par curiosité et simplicité de lecture / écriture, est-ce possible de l'écrire comme ça ?

```suggestion
                <Suspense fallback={<LoadingPage />}>
                  <CheatcodesStackNavigator />
                </Suspense>
```

Reply:
Çe ne semble pas possible car l'utilisation de {() => (...)} garantit que la fonction enfant recevra les props de navigation nécessaires, car React Navigation appelle cette fonction et passe automatiquement les props, comme navigation, route, etc. Le typage aussi râle : 

```
Impossible d'assigner le type 'Element' au type '(props: { route: RouteProp<RootStackParamList, "CheatcodesStackNavigator">; navigation: any; }) => ReactNode'.
  Le type 'Element' ne fournit aucune correspondance pour la signature '(props: { route: RouteProp<RootStackParamList, "CheatcodesStackNavigator">; navigation: any; }): ReactNode'.ts(2322)
types.d.ts(325, 5): Le type attendu provient de la propriété 'children', qui est déclarée ici sur le type 'IntrinsicAttributes & RouteConfig<RootStackParamList, "CheatcodesStackNavigator", StackNavigationState<ParamListBase>, StackNavigationOptions, StackNavigationEventMap>'
```